lib/class-wp-rest-blocks-renderer-controller.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/**
+ * Blocks Renderer REST API: WP_REST_Blocks_Renderer_Controller class
+ *
+ * @package gutenberg
+ * @since ?
+ */
+
+/**
+ * Controller which provides REST endpoint for rendering blocks.
+ *
+ * @since ?
+ *
+ * @see WP_REST_Controller
+ */
+class WP_REST_Blocks_Renderer_Controller extends WP_REST_Controller {
```

Comment:
Should this be singular? `Block_Renderer...`

Reply:
Yah, same with the endpoint route.

Reply:
Done in c4abc69